packages/amplify-migration-tests/src/configure/index.ts

```diff
@@ -0,0 +1,51 @@
+import * as nexpect from 'nexpect';
```

Comment:
We are no longer using nexpect from NPM, use the one which in amplify-e2e-core

Reply:
Using e2e-core